pkg/database/mysql/deploy.go

```diff
@@ -104,6 +113,13 @@ func buildColumnStatements(m *MysqlConnection, tableName string, mysqlTableSchem
 			return nil, err
 		}
 
+		if isParameterizedColumnType(dataType) {
+			dataType, err = maybeParseParameterizedColumnType(dataType)
```

Comment:
we would always generate alter statements because an integer type in the db would return `int` here but from the schema it would be `int (11)`